wls-common/src/main/java/org/jboss/arquillian/container/wls/CommonWebLogicConfiguration.java

```diff
@@ -527,4 +529,15 @@ public boolean isDeployExplodedArchive() {
    public void setDeployExplodedArchive(boolean deployExplodedArchive) {
        this.deployExplodedArchive = deployExplodedArchive;
    }
+
+   /**
+    * Remote machine configuration flag. Specify whether the target server is located on another machine (ie. if the
+    * target server do not share the same file system as the test server).
+    *
+    * @return {@code true} if executing test on a remote machine, {@code false} otherwise.
+    */
+   // Note: getter-only, setter must be implemented in child classes.
+   public boolean isRemoteMachine() {
+      return remoteMachine;

```

Comment:
Actually, testing the adminUrl may not always provide a satisfying result if the server is running on the local host but is somehow on a isolated filesystem (eg. if the target server is in a Docker container). In such case, you may want to force the remoteMachine flag as well.

If we want to make the handling of this flag more transparent to the end-user, I would suggest that we either:
- Test for the adminUrl first and then fallback to the remoteMachine property.
- Always enable the "-remote" argument for the remote adapters...